src/site/content/en/blog/yahoo-japan-news/index.md

```diff
@@ -0,0 +1,216 @@
+  <figcaption class="w-figcaption">
+    Lighthouse "Avoid large layout shifts" audit shows which elements are
+    contributing to CLS score and how much.
+  </figcaption>
+</figure>
+
+It's important to keep in mind the _cumulative_ part of the Cumulative Layout
+Shift—the score is captured through the entire page lifecycle. In the
+real-world, the score can include shifts that happen as a result of user
+interactions such as scrolling a page or tapping a button. To collect CLS scores
+from the [field
+data](https://web.dev/how-to-measure-speed/#lab-data-vs-field-data), the team
+integrated [web-vitals](https://github.com/GoogleChrome/web-vitals/) JavaScript
+library reporting.
+
+{% Aside %}  
+As a part of performance monitoring strategy, they're also working on building
+an internal tool with [Lighthouse CI](https://web.dev/lighthouse-ci/) to
+continuously audit performance across businesses in the company.  
+{% endAside %}
+
+The team used Chrome DevTools to identify which elements were making layout
+shifts on the page.
+[Layout Shift Regions](https://developers.google.com/web/updates/2019/07/devtools)
+in DevTools visualizes elements that contribute to CLS by highlighting them with
+a blue rectangle whenever a layout shift happens. 
+
+<figure class="w-figure">
+    {% Img src="image/vgdbNJBYHma2o62ZqYmcnkq3j0o1/scvRgLxkZVQoGfOtjZ4z.png", alt="Article details page with blue rectangles overlayed on the hero image and the text.", class="w-screenshot", width="376", height="668" %}
+  <figcaption class="w-figcaption">
+    Visualized layout shifts.
+  </figcaption>
+</figure>
+
+They figured out that a layout shift occurred after the hero image at the top of
+the article was loaded for the first view. 
+
+<figure class="w-figure">
+    {% Img src="image/vgdbNJBYHma2o62ZqYmcnkq3j0o1/HdGiCh7O8ZhqcOkfVGEv.jpg", alt="Screenshots of the article details page showing side by side comparison before and after layout shift.", class="w-screenshot", width="798", height="455" %}
+  <figcaption class="w-figcaption">
+    Layout shift on the article detail page.
+  </figcaption>
+</figure>
+
+
+In the example above, when the image finishes loading, the text gets pushed down
+(the position change is indicated with the red line).
+
+## Improving CLS for images
+
+For fixed-size images, layout shifts can be prevented by specifying the `width`
+and `height` attributes in the `img` element and using the CSS
+[aspect-ratio](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/aspect-ratio)
+property available in modern browsers. However, Yahoo! JAPAN News needed to
+implement images that also meet the following requirements:
+
+-  Prevent the aspect ratio from collapsing.
+-  Display images of various sizes (such as portraits and landscapes).
+-  Display images at 100% width of the parent element. However, images with
+   height over 300 px will be scaled down to 300 px height while preserving the
+   aspect ratio.
+-  Images can have links and be clickable.
```

Comment:
```suggestion
-  Make images clickable and support links.
```
@mihajlija check out the writing concept of parallelism: https://www.grammarly.com/blog/parallelism/

In this case all the other list items start with a verb so we can make this one more parallel by also starting with a verb